File: cs/Iris/ParserUtils/LexParseException/UnfinishedString.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Iris.ParserUtils.LexParseException
{
    public class UnfinishedString : BaseException
    {
        public UnfinishedString(int lineNum, int colNum)
            : base(lineNum, colNum)
        {

        }

        public override string ExceptionToString()
        {
            StringBuilder str = new StringBuilder();

            str.Append("unfinished string: started at ");
            str.Append(base.stringTrace());

            return str.ToString();
        }
    }
}

File: java/Iris/src/com/rokuan/iris/exception/UnfinishedString.java
package com.rokuan.iris.exception;

public class UnfinishedString extends BaseException
{
	public UnfinishedString(int lineNum, int colNum)
	{
		super(lineNum, colNum);
	}

	public String ExceptionToString()
	{
		StringBuilder str = new StringBuilder();

		str.append("unfinished string: started at ");
		str.append(super.stringTrace());

		return str.toString();
	}
}
